cmake/EDM4HEPConfig.cmake.in

```diff
@@ -8,6 +8,9 @@
 # the imported targets created later.
 set_and_check(EDM4HEP_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
 
+# - Create path to installed read-only data files (e.g. yaml files)
+set_and_check(EDM4HEP_DATA_DIR "@PACKAGE_CMAKE_INSTALL_DATADIR@/edm4hep")
```

Comment:
If I understood correctly, this would be `<prefix>/share/edm4hep` in the end (assuming "standard" install settings)? Any reason to not put this into `<prefix>/share` directly? Mainly asking because Key4hep package *environment* variables usually point to `<prefix>/share` when installed via `spack`.

Reply:
> "you should install your data in a subdirectory" 

Ah, just saw that part. I suppose that is the reason for the subdirectory.
Also, I just checked a few other Key4hep packages, and they seem to actually also put things into `<prefix>/share/<package>`.

Apologies for the noise, I blame Monday morning ;)